quantum/keycode_config.c

```diff
@@ -122,40 +122,38 @@ __attribute__((weak)) uint16_t keycode_config(uint16_t keycode) {
  */
 
 __attribute__((weak)) uint8_t mod_config(uint8_t mod) {
+    /**
+     * Note: The right hand MOD variables (MOD_RALT, MOD_RGUI, MOD_RSFT, MOD_RCTL)
```

Comment:
Might be simpler to mention that these are for the 5-bit packed mods, not the full, 8-bit mods.  And more info about the mods can be seen in `modifiers.h`.